packages/plugin-redirects/README.md

```diff
@@ -2,6 +2,163 @@
 
 A plugin for [Payload](https://github.com/payloadcms/payload) to easily manage your redirects from within your admin panel.
 
+## Features
+
+- Manage redirects directly from your admin panel
+- Support for internal (reference) and external (custom URL) redirects
+- Built-in multi-language support
+- Optional redirect types (301, 302, etc.)
+
+## Installation
+
+```bash
+pnpm add @payloadcms/plugin-redirects
+```
+
+## Basic Usage
+
+In your [Payload Config](https://payloadcms.com/docs/configuration/overview), add the plugin:
+
+```ts
+import { buildConfig } from 'payload'
+import { redirectsPlugin } from '@payloadcms/plugin-redirects'
+
+export default buildConfig({
+  plugins: [
+    redirectsPlugin({
+      collections: ['pages'], // Collections to use in the 'to' relationship field
+    }),
+  ],
+})
+```
+
+## Configuration
+
+### Options
+
+| Option                      | Type       | Description                                                                                             |
+| --------------------------- | ---------- | ------------------------------------------------------------------------------------------------------- |
+| `collections`               | `string[]` | An array of collection slugs to populate in the `to` field of each redirect.                            |
+| `overrides`                 | `object`   | A partial collection config that allows you to override anything on the `redirects` collection.         |
+| `redirectTypes`             | `string[]` | Provide an array of redirects if you want to provide options for the type of redirects to be supported. |
+| `redirectTypeFieldOverride` | `Field`    | A partial Field config that allows you to override the Redirect Type field if enabled above.            |
+
+### Advanced Example
+
+```ts
+import { buildConfig } from 'payload'
+import { redirectsPlugin } from '@payloadcms/plugin-redirects'
+
+export default buildConfig({
+  plugins: [
+    redirectsPlugin({
+      collections: ['pages', 'posts'],
+
+      // Add custom redirect types
+      redirectTypes: ['301', '302'],
+
+      // Override the redirects collection
+      overrides: {
+        slug: 'custom-redirects',
+
+        // Add custom fields
+        fields: ({ defaultFields }) => {
+          return [
+            ...defaultFields,
+            {
+              name: 'notes',
+              type: 'textarea',
+              admin: {
+                description: 'Internal notes about this redirect',
+              },
+            },
+          ]
+        },
+      },
+    }),
+  ],
+})
+```
+
+## Custom Translations
+
+The plugin automatically includes translations for English, French, and Spanish. If you want to customize existing translations or add new languages, you can override them in your config:
```

Comment:
I noticed that other plugins, like eCommerce, only support English. I plan to add all the plugins to our translation script later on to include all the admin panel languages.